docs/UserManuals/ConfigUI/AdvancedMode.md

```diff
@@ -63,18 +66,20 @@ Below is an example for collecting 2 GitLab repos sequentially. It has 2 stages,
 ```
 
 ### 2. Github
+
 Collect a GitHub repo and a Jira board in parallel
```

Comment:
Remove the Jira part in this section and update it to 'collect multiple GitHub repos'?